scripts/generate_data_in_parallel.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
+# gracefully exit
+trap "cleanup" SIGINT SIGTERM EXIT
+
 # Usage: generate_data_in_parallel.sh <relative config_path> <num_files_per_process> <num_processes> <num_workers_per_process>
```

Comment:
`# Usage: generate_data_in_parallel.sh <relative config_path> <num_files_per_process> <num_processes> <num_workers_per_process>`
Can you update this comment to reflect the new usage of the script?

Reply:
Done.